README.md

```diff
@@ -1,10 +1,39 @@
-# liphdbpp
-Interface library for HDB++
-
-## building
-git clone --recursive http://github.com/tango-controls/libhdbpp.git  
-cd libhdbpp  
-export TANGO_DIR=/usr/local/tango-9.2.5a  
-export OMNIORB_DIR=/usr/local/omniorb-4.2.1  
-export ZMQ_DIR=/usr/local/zeromq-4.0.7  
+# Libhdbpp
+
+Interface library for the HDB++ archiving system. Libhdbpp provides an abstract means to archive data events to either a Cassandra or MySQL database.
```

Comment:
I would even add something like "and potentially other database back-ends".

Reply:
Added